testsuite/features/step_definitions/retail_steps.rb

```diff
@@ -91,27 +91,54 @@
   end
 end
 
-Then(/^terminal "([^"]*)" should have got a retail network IP address$/) do |host|
+Then(/^"([^"]*)" should communicate with the server using public interface/) do |host|
```

Comment:
Then xxx **host** should yada yada

Reply:
It will go into next PR anyway, I will restrict the scope of this PR